lua/net.lua

```diff
@@ -52,6 +52,12 @@ return function (gatekeeper_server)
 	-- How often the log file should be rotated. The unit is second.
 	local rotate_log_interval_sec = 60 * 60 -- 1h
 
+	-- The user that will run Gatekeeper after it boots.
+	local user = "gatekeeper"
```

Comment:
Recommend that we either change the user in the path of the rte_kni module in `lua/cps.lua` (which is currently `"user"`) to `"gatekeeper"` to match this or change this variable to be `"user"` as well.

Reply:
Let's update the path in `lua/cps.lua`.

Reply:
Updated.